angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.ts

```diff
@@ -44,34 +56,43 @@ import { ISprkDropdownChoice } from './sprk-dropdown.interfaces';
       <div [ngClass]="getClasses()" *ngIf="isOpen">
         <div
           class="sprk-c-Dropdown__header"
-          *ngIf="dropdownType === 'mastheadSelector' || title || selector"
+          *ngIf="
+            this.getVariant() === 'mastheadSelector' ||
+            returnSecondIfBoth(title, heading) ||
+            selector
+          "
         >
-          <h2 class="sprk-c-Dropdown__title sprk-b-TypeBodyTwo" *ngIf="title">
-            {{ title }}
+          <h2
+            class="sprk-c-Dropdown__title sprk-b-TypeBodyTwo"
+            *ngIf="title || heading"
+          >
+            {{ returnSecondIfBoth(title, heading) }}
           </h2>
 
           <a
             sprkLink
-            *ngIf="selector && !title"
+            *ngIf="selector && (!title || !heading)"
             variant="plain"
             class="sprk-o-Stack sprk-o-Stack--split@xxs sprk-o-Stack--center-column sprk-u-Width-100"
             (click)="toggle($event)"
-            [attr.aria-label]="title"
+            [attr.aria-label]="selector"
```

Comment:
Found this mistake in the middle of this task: this element never renders if `!title` so the `aria-label` would never show up. `selector makes more sense here.

Reply:
Good catch